frontend/src/hooks/useDb.ts

```diff
@@ -15,6 +15,13 @@ class GenLayerSimulatorDB extends Dexie {
       transactions:
         '++id, type, status, contractAddress, localContractId, txId',
     });
+
+    this.version(3).stores({
+      contractFiles: 'id',
+      deployedContracts: '[contractId+address]',
+      transactions:
+        '++id, type, status, contractAddress, localContractId, hash',
+    });
```

Comment:
what's this? I'm not knowledgeable on the frontend :D

Reply:
This is an updated model for the local IndexedDB to tell it that there's now a hash column.